Improve the links meaning for the Accessibility and SEO. (#32107)
mdn/content: files/en-us/learn/javascript/first_steps/variables/index.md

The square brackets specify an index value corresponding to the position of the value you want returned. You might have noticed that arrays in JavaScript are zero-indexed: the first element is at index 0.

> To learn more, see our article on [arrays](/en-US/docs/Learn/JavaScript/First_steps/Arrays).

Reviewer: ```suggestion
To learn more, see our article on [Arrays](/en-US/docs/Learn/JavaScript/First_steps/Arrays).
```

> You'll learn a lot more about arrays in [the arrays article](/en-US/docs/Learn/JavaScript/First_steps/Arrays).

Reviewer: To avoid repetition, consider this:
```suggestion
To learn more, see the [Arrays](/en-US/docs/Learn/JavaScript/First_steps/Arrays) page.
```

Author: If we can make the link label more meaningful, I think that will be more better for the browsers and screen readers tools like when a blind person wants to know just the links in the page quickly.
`To learn more, see the [Arrays Article](/en-US/docs/Learn/JavaScript/First_steps/Arrays).`

Reviewer: @Amer-Adel the link text should match the title of the page. This is the recommended practice for accessibility, especially to help those using screen readers to convey where the link will take them.

For this particular instance, there is another option:
```suggestion
To learn more, see our page on [Arrays](/en-US/docs/Learn/JavaScript/First_steps/Arrays).
```

Author: @dipikabh thank you for letting me know about this. I am just taking into consideration someone who only heard the title among the rest of the links quickly. Will he know if this is a link to an article, an example, or a test?

Reviewer: @Amer-Adel We're saying "page". You could change that to article if that helps

```suggestion
To learn more, see our article on [Arrays](/en-US/docs/Learn/JavaScript/First_steps/Arrays).
```

Author: @dipikabh What I mean is to put this word, whatever it is😅, inside the link title along with the 'arrays' to explain it, so that it is readable to blind people using screen readers, as I explained in the [example above](https://github.com/mdn/content/pull/32107#discussion_r1509920226).

Reviewer: @Amer-Adel Check some of our other guidelines about link texts in the sections [Cross-references](https://developer.mozilla.org/en-US/docs/MDN/Writing_guidelines/Writing_style_guide#cross-references_linking) and [Link text](https://developer.mozilla.org/en-US/docs/MDN/Writing_guidelines/Writing_style_guide#link_text); check out the examples. In the link text, we don't mention whether it is a section or an article. The link text should either match the article/section title or be a descriptive phrase.

Reviewer: Glad you've reached that understanding and checked out the style guide.

dog.name;
```

> We won't be looking at objects any more for now — you can learn more about those in [the objects module](/en-US/docs/Learn/JavaScript/Objects).

Reviewer: ```suggestion
For more on this topic, see the [Introducing JavaScript objects](/en-US/docs/Learn/JavaScript/Objects) module.
```
